pkg/collector/corechecks/system/file_handles.go

```diff
@@ -3,6 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-2020 Datadog, Inc.
 // +build !windows
+// +build !freebsd
```

Comment:
@mx-psi would this line needed changes as well to be `// +build !windows !freebsd`? I wasn't aware about the logic if used in multiple lines.

Reply:
I don't think a change is needed here: that means `!windows AND !freebsd`, which is what we want (we want to exclude this from both of those platforms). Multiple lines are joined by an `AND`